From: Markus Koschany <apo@debian.org>
Date: Sat, 3 Oct 2020 18:13:24 +0200
Subject: platform libs.batik.read

Batik module is currently disabled. Enable it when an application actually
needs it?
---
 platform/libs.batik.read/nbproject/project.properties | 16 ----------------
 1 file changed, 16 deletions(-)

diff --git a/platform/libs.batik.read/nbproject/project.properties b/platform/libs.batik.read/nbproject/project.properties
index 870874f..510b8f5 100644
--- a/platform/libs.batik.read/nbproject/project.properties
+++ b/platform/libs.batik.read/nbproject/project.properties
@@ -31,22 +31,6 @@ file.reference.batik-util-1.12.jar=external/batik-util-1.12.jar
 file.reference.batik-xml-1.12.jar=external/batik-xml-1.12.jar
 file.reference.xml-apis-ext-1.3.04.jar=external/xml-apis-ext-1.3.04.jar
 file.reference.xmlgraphics-commons-2.4.jar=external/xmlgraphics-commons-2.4.jar
-release.external/batik-anim-1.12.jar=modules/ext/batik-anim-1.12.jar
-release.external/batik-awt-util-1.12.jar=modules/ext/batik-awt-util-1.12.jar
-release.external/batik-bridge-1.12.jar=modules/ext/batik-bridge-1.12.jar
-release.external/batik-constants-1.12.jar=modules/ext/batik-constants-1.12.jar
-release.external/batik-css-1.12.jar=modules/ext/batik-css-1.12.jar
-release.external/batik-dom-1.12.jar=modules/ext/batik-dom-1.12.jar
-release.external/batik-ext-1.12.jar=modules/ext/batik-ext-1.12.jar
-release.external/batik-gvt-1.12.jar=modules/ext/batik-gvt-1.12.jar
-release.external/batik-i18n-1.12.jar=modules/ext/batik-i18n-1.12.jar
-release.external/batik-parser-1.12.jar=modules/ext/batik-parser-1.12.jar
-release.external/batik-script-1.12.jar=modules/ext/batik-script-1.12.jar
-release.external/batik-svg-dom-1.12.jar=modules/ext/batik-svg-dom-1.12.jar
-release.external/batik-util-1.12.jar=modules/ext/batik-util-1.12.jar
-release.external/batik-xml-1.12.jar=modules/ext/batik-xml-1.12.jar
-release.external/xml-apis-ext-1.3.04.jar=modules/ext/xml-apis-ext-1.3.04.jar
-release.external/xmlgraphics-commons-2.4.jar=modules/ext/xmlgraphics-commons-2.4.jar
 
 is.autoload=true
 javac.source=1.6
